apps/cash-register/public/script.js

```diff
@@ -101,18 +105,24 @@ const updateUI = change => {
   };
   // Update cid if change is passed in
   if (change) {
-    change.forEach(changeArr => {
-      const targetArr = cid.find(cidArr => cidArr[0] === changeArr[0]);
-      targetArr[1] = parseFloat((targetArr[1] - changeArr[1]).toFixed(2));
+    change.forEach(([changeDenomination, changeAmount]) => {
+      const targetArr = cid.find(
+        ([denominationName, _]) => denominationName === changeDenomination
```

Comment:
```suggestion
        ([denominationName]) => denominationName === changeDenomination
```

This one's not necessary.